dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -169,8 +167,19 @@ public CollectionResource wrapResource(CollectionRest collection, String... rels
     }
 
     @Override
-    @PreAuthorize("hasAuthority('ADMIN')")
     protected CollectionRest createAndReturn(Context context) throws AuthorizeException {
+        throw new DSpaceBadRequestException("Missing parent parameter.");
```

Comment:
While this error message is accurate, it looks odd in this class (as there's no `parent` param anywhere in the `CollectionRestRepository` anymore.  So, I think we should reword this slightly to say something like:
`"Cannot create a Collection without providing a parent Community"`